x-pack/plugins/infra/server/lib/domains/log_entries_domain/builtin_rules/filebeat_mysql.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const mysqlRules = [
+  {
+    when: {
+      exists: ['mysql.error.message'],
```

Comment:
As a heads up: I hope in 7.0 all modules will have `message` fields. So this mainly applies to 6.x data.

@weltenwort To cover in 7 data from 6 and 7, these rules will just have addition `when` parts? Long term would be great if these rules could even be adjusted by the users if needed.

Reply:
I would say we add rules for both the 6.x format and the 7.x (because users might want to view older logs). As longs as the conditions are specific enough, they can coexist.

Making the rules adjustable via the UI is definitely planned. That's why they are written in this verbose declarative format.

Reply:
I agree with @weltenwort, we should keep the rules for 6.x